docs/_artifacts/document-attributes.adoc

```diff
@@ -3,41 +3,41 @@
 // Attributes MUST appear below the document title
 // Do not delete attributes with identical values, such as :product-long: and :product:. These are intentional because although they are identical here, they are not in other renderings of this source file.
 
-
+//General
 :org-name: Application Services
 :community:
 :imagesdir: ./images
-:product-version: 1
-:product-long: OpenShift Streams for Apache Kafka
-:product: Streams for Apache Kafka
-:registry-product-long: OpenShift Service Registry
-:registry-product: Service Registry
-:registry: Service Registry
 :cli-product-long: OpenShift Application Services
-// Placeholder URL, when we get a HOST UI for the service we can put it here properly
-:service-url: https://console.redhat.com/application-services/streams/
-:registry-url: https://console.redhat.com/application-services/service-registry/
 :property-file-name: app-services.properties
-
-// Other upstream project names
 :samples-git-repo: https://github.com/redhat-developer/app-services-guides
-
-//URL components for cross refs
-:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
+:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/docs/
 :base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
-:getting-started-url: getting-started/README.adoc
-:getting-started-service-registry-url: getting-started-service-registry/README.adoc
-:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
-:kafkacat-url: kcat/README.adoc
-:quarkus-url: quarkus/README.adoc
-:quarkus-service-registry-url: quarkus-service-registry/README.adoc
 :rhoas-cli-url: rhoas-cli/README.adoc
```

Comment:
@wtrocki @bhardesty I'm not sure if the intention is to have rhoas cli be it's own product with it's own directory, or just have all CLI content simply at the parent level alongside other products for common access. So either way, left all CLI base attributes under this general section for now.

Reply:
I think the CLI content should be in a directory of its own -- at least two guides (the Command reference and the Getting started) should be included there. A separate "cli" directory would keep the overall directory structure less cluttered.

Reply:
@bredamc I agree. I was thinking of just a `rhoas` directory and can contain CLI content and anything else generically for overall rhoas. Wdyt?

Reply:
@sterobin I see a `rhoas-cli` directory has actually been added. I wondered if the "rh" should be omitted, but I guess that's what the command is called :)

Reply:
Yeah, I remember some discussion about the fact that even the upstream component is `rhoas` instead of just `oas` :) or whatever. But that's an ancient conversation and I don't know where it ended up. But naming the directory the same as that component is all I suggest, which as of now is still `rhoas`.

Reply:
@sterobin I would include rhoas cli as its own product in its own directory. I think we'll likely pursue this same strategy downstream as well.

Reply:
@bredamc @bhardesty I pushed a commit to add a `rhoas` product folder, which houses the rhoas cli doc(s) and any future general rhoas resources, such as Operator documentation. I can envision that applying for both the CLI and Operator alike, so I thought it wise to have a more generic rhoas product folder, rather than a CLI product folder. Let me know what you think.